desc/backend.py

```diff
@@ -10,15 +11,23 @@
 from desc import config as desc_config
 from desc import set_device
 
+verbose = True
+
+# set child processes to use numpy backend and suppress print statements
+if not multiprocessing.current_process().name == "MainProcess":
```

Comment:
can you explain again why we have to use numpy in the child processes? I feel like that could be a significant slowdown since all the jax loops will revert to python loops.

Also, could this break user code that is running DESC in multiple processes?

Reply:
JAX is not compatible with python multithreading, and I think it's because JAX is already multithreaded itself so that leads to conflicts. But only the spawned processes are using Numpy, the main code should still be using JAX. 

There might be ways to get around this that we could look into: https://github.com/google/jax/issues/743

But in general, I think the most common use case is to write a wout file and then call some Fortran code on that input. And you can pre-compute all the transforms with JAX in the main code and then cast them to Numpy arrays to pass to the external function.

Reply:
I'm not sure about your second question of breaking code that is running DESC in multiple processes. I'm having issues using these external objectives when running on a GPU, even though the same code runs fine on a CPU. Maybe that is related?

Reply:
check local imports of jax for possible issues

Reply:
Try w/o multiprocessing on GPU and see if wout computation is faster